src/styles/main.scss

```diff
@@ -1 +1,77 @@
-// put code here
+@import 'variables';
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+
+  display: flex;
+  flex-direction: column;
```

Comment:
The flex-direction property is set to column, which is not necessary in this case as there's only one child element.